src/python/turicreate/test/test_linear_regression.py

```diff
@@ -25,6 +25,7 @@
 
 try:
     import statsmodels.formula.api as sm
+    from sklearn import linear_model
```

Comment:
`scikit-learn` is included in our `scripts/requirements.txt` file. So I think getting an import error would be unexpected and we shouldn't be ignoring it. Indeed ignoring it here is just going to cause a more confusing exception later.

I think removing this import statement from the middle of a function is good. I would probably just move it to the top of the function in which it is used.

Reply:
That's used for minimal pkg. For minimal pkg, its test env doesn't have sklearn. But pytest will run all test files and do the filtering. So I need to add this.

Reply:
I don't think pytest will run all tests. I think it will import them, but it's not going to run them all. If you move the import to the top of the one function it is used in, then you will not have this issue.

Reply:
I know. To be precise, `run all test files` -> load all test files.

If you only have a handy of linear_model, I would do what you suggest. I'm lazy to replace each one of them into `from sklearn import linear_model`.

Reply:
Oh, ok I thought the import statement you removed was the only place it was getting used.